upgrade/upgrade-scripts/verify-jar

```diff
@@ -18,11 +18,16 @@
 # shellcheck disable=SC1090
 . "${BASH_SOURCE%/*}/common.sh"
 
-IMAGE_VERSION=$(get_image_version)
-[[ -n "$IMAGE_VERSION" ]] || die "failed to determine image version"
+# We source the version information to bring in values from
+# the version.info file, such as
+#   - VERSION : the appliance version
+#   - HOTFIX : the hotfix version if any
+
+source_version_information
+
+[[ -n "$VERSION" ]] || die "failed to determine image version"
```

Comment:
This is unnecessary, right? If so, how do you feel about removing it? IIRC, this same check is already done by the `source_version_information` function.